content/blog/2019-__-__-hydrology_task_view.md

```diff
@@ -0,0 +1,153 @@
+---
+slug: hydrology-task-view
+title: "Getting your toes wet in R: Hydrology, meteorology, and more"
+authors:
+- name: Sam Albers
+  url: https://github.com/boshek
+- name: Ilaria Prosdocimi
+  url: https://github.com/ilapros
+- name: Sam Zipper
+  url: https://github.com/samzipper
+date: "foo"
+categories: technotes
+topicid:
+tags:
+- r
+- packages
+- hydrology
+- cran
+output:
+  html_document:
+    keep_md: yes
+---
+
+
+
+## Importance of Hydrology
+Given that liquid water is [essential to life on Earth](https://www.amnh.org/explore/science-topics/water-and-life-on-earth), water research cuts across numerous disciplines including hydrology, meteorology, geography, climate science, engineering, ecology, and more. Numerous R packages have emerged from this diversity of approaches, and we recently gathered many of these packages into a new rOpenSci [task view](https://github.com/ropensci/Hydrology) which we broadly titled 'Hydrology' and published to [CRAN](https://CRAN.R-project.org/view=Hydrology). Our intent is to be exhaustive and compile R packages to access, model, and summarise information related to the movement of water across the Earth's landscape. We hope to contribute to a [nascent community of hydrological R users](https://www.hydrol-earth-syst-sci-discuss.net/hess-2019-50/) and develop an infrastructure of packages that provide a comprehensive toolkit for water practitioners who use R as their preferred computational analysis tool. Making sense of water data is critical to understanding the response of landscapes to a changing climate. Consolidating water-related packages will promote their usage and discovery and ultimately facilitate reproducible workflows for water research. Since this is a new task view, it serves our purpose to evaluate the current **State of Hydrology** in R and look at the interdependency of hydrology packages relative to some better known collections of packages in the general R ecosystem. 
```

Comment:
consider "many of these packages" -> "many of them" 
and/or because lots of use of word "packages", see if it can be minimized. Tough challenge. Fine as is